Cargo.toml

```diff
@@ -1,73 +1,20 @@
-[package]
-name = "aa-bundler"
-version = "0.1.0"
-authors = ["Vid Kersic <vid.kersic@yahoo.com>"]
-edition = "2021"
-description = "AA - Bundler implementation"
-default-run = "bundler"
-rust-version = "1.67.1"
-
-[dependencies]
-anyhow = "1"
-arrayref = "0.3"
-async-trait = "0.1"
-clap = { version = "4", features = ["derive"] }
-dashmap = "5.4.0"
-dirs = "4.0"
-educe = { version = "0.4", features = ["Debug", "Default"] }
-ethereum-interfaces = { git = "https://github.com/ledgerwatch/interfaces" }
-ethers = { version = "2.0.1", features = ["solc-full"] }
-ethers-providers = "2.0.1"
-expanded-pathbuf = "0.1"
-hex = { version = "0.4.3", default-features = false, features = ["std"] }
-jsonrpsee = { version = "0.16", features = ["server", "macros"] }
-lazy_static = "1.4.0"
-page_size = "0.5.0"
-parking_lot = "0.12"
-prost = "0.11"
-regex = "1"
-reth-db = { git = "https://github.com/paradigmxyz/reth.git", rev = "aa6f2cb0610fb4fa0926b42cfed7f8ff51e0db8a" }
-reth-libmdbx = { git = "https://github.com/paradigmxyz/reth.git", rev = "aa6f2cb0610fb4fa0926b42cfed7f8ff51e0db8a" }
-ron = "0.8"
-rustc-hex = "^2.0.1"
-serde = "1"
-serde_json = "1"
-thiserror = "1"
-tokio = { version = "1.18", features = ["full"] }
-tonic = { version = "0.8", default-features = false, features = [
-    "codegen",
-    "prost",
-    "transport",
-] }
-tracing = "0.1"
-tracing-subscriber = "0.3"
-
-[dev-dependencies]
-tempdir = "0.3.7"
-
-[build-dependencies]
-anyhow = "1"
-ethers = { version = "2.0.1", features = ["solc-full"] }
-prost-build = "0.11"
-protobuf-src = "1.1.0"
-tonic-build = "0.8"
-
-[[bin]]
-path = "bin/bundler.rs"
-name = "bundler"
-
-[[bin]]
-path = "bin/bundler-uopool.rs"
-name = "bundler-uopool"
-
-[[bin]]
-path = "bin/bundler-rpc.rs"
-name = "bundler-rpc"
-
-[[bin]]
-path = "bin/create-wallet.rs"
-name = "create-wallet"
-
+[workspace]
+members = [
+    "bin/bundler",
+    "crates/bundler",
+    "crates/contracts",
+    "crates/grpc",
+    "crates/primitives",
+    "crates/rpc",
+    "crates/uopool",
+    "tests",
+]
+default-members = ["bin/bundler"]
+
+[profile.debug-fast]
```

Comment:
I am curious what this means. I could not google any good results on it. Maybe you could provide some links or description?